sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala

```diff
@@ -141,8 +140,7 @@ class HiveSparkSubmitSuite
     runSparkSubmit(args)
   }
 
-  test("SPARK-8020: set sql conf in spark conf") {
-    assume(!SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9))
+  ignore("SPARK-8020: set sql conf in spark conf") {
```

Comment:
Will there be a chance for these ignored cases to be restarted?

Reply:
shall we change the hive version in these tests?

Reply:
For this and and the last one, we could. But the others don't set the Hive version apparently. I will just take a look separately.